pkg/web/apps/apps_psql_repository_test.go

```diff
@@ -45,6 +45,18 @@ var (
 		WHERE app_id=\$1;`
 )
 
+type AnyTimestamp struct{}
+
+func (e AnyTimestamp) Match(v driver.Value) bool {
+	return true
+}
+
+type AnyUUID struct{}
```

Comment:
See here that the UUID can be generated in the class. In the test to ignore the mock value, you can use this.